api/v1beta1/azuremanagedcontrolplane_types.go

```diff
@@ -190,12 +208,11 @@ type AKSSku struct {
 }
 
 // LoadBalancerProfile - Profile of the cluster load balancer.
+// At most one of `managedOutboundIPs`, `outboundIPPrefixes`, or `outboundIPs` may be specified.
+// See also [AKS doc].
+//
+// [AKS doc]: https://learn.microsoft.com/azure/aks/load-balancer-standard
 type LoadBalancerProfile struct {
-	// Load balancer profile must specify at most one of ManagedOutboundIPs, OutboundIPPrefixes and OutboundIPs.
-	// By default the AKS cluster automatically creates a public IP in the AKS-managed infrastructure resource group and assigns it to the load balancer outbound pool.
-	// Alternatively, you can assign your own custom public IP or public IP prefix at cluster creation time.
-	// See https://docs.microsoft.com/en-us/azure/aks/load-balancer-standard#provide-your-own-outbound-public-ips-or-prefixes
```

Comment:
Is this comment no longer accurate?

Reply:
I removed this for the sake of not keeping too many of the finer AKS details in our docs. That's all described in the linked AKS docs. I think a good rule of thumb is to keep enough detail here for users to know when they should dig into the AKS docs without reproducing all that content here.